event/event.go

```diff
@@ -0,0 +1,181 @@
+// event is a package that models events that occur during the execution of a program
```

Comment:
Add a license comment: https://github.com/hajimehoshi/ebiten/blob/master/CONTRIBUTING.md

Reply:
OK, I will do this.